### ISSUE TITLE
Add a Floyd–Warshall algorithm example

### ISSUE THREAD

**Issue by kylelutz** [boostorg/compute#88]
Add an example named `floyd_warshall` implementing the [Floyd–Warshall](http://en.wikipedia.org/wiki/Floyd%E2%80%93Warshall_algorithm) all-pairs shortest path algorithm on a graph represented by an adjacency-matrix storing the edge weights.
##

**Comment by ya1gaurav** [comment]
This example is related with Boost Compute ?

**Comment by kylelutz** [comment]
Yeah, it would be good to have an example demonstrating how to use the Boost.Compute API to implement a basic graph algorithm like all-pairs shortest paths.

**Comment by ya1gaurav** [comment]
Ok, I will work on this. I have used this algorithm in programming problems. It will be new experience with Boost Compute.

**Comment by kylelutz** [comment]
Sounds great! Thanks! Let me know if you run into any issues or have any questions.

**Comment by ya1gaurav** [comment]
Sure, it will take me some time as I am beginner to boost compute. I will see some boost compute examples to start with.
